vyper/venom/passes/dft.py

```diff
@@ -1,14 +1,19 @@
 from vyper.utils import OrderedSet
-from vyper.venom.analysis import DFG
+from vyper.venom.analysis.dfg import DFGAnalysis
```

Comment:
it will be nice to have `vyper/venom/analysis/__init__.py` which exposes the classes, so we don't need to import each class from the specific module